src/Ledger/Utxo.lagda

```diff
@@ -160,6 +160,9 @@ the state of the previous era at the transition into the Conway era.
 Alternatively, we can effectively treat the old handling of deposits
 as an erratum in the Shelley specification, which we fix by implementing
 the new deposits logic in older eras and then replaying the chain.
+(The handling of deposits in the Shelley era is discussed
+in~\cite[Sec.~8]{shelley-ledger-spec}
+and~\cite[Sec.~B.2]{shelley-delegation-design}.)
```

Comment:
Another rambling. Using biblatex this would be:
```latex
\texcites[Sec.~8]{shelley-ledger-spec}[App.~B.2]{shelley-delegation-design}
```

Reply:
Good note for the PR that addresses #681.